utilities/pytest_utils.py

```
@@ -347,6 +348,14 @@ def wait_for_pod_deletion(pod):
                         leftovers.setdefault(vmware_provider.type, []).append({
                             "cloned_vm_name": _cloned_vm_name,
                         })
+
+                for _vmdk in orphaned_vmdks:
+                    _vmdk_path = _vmdk["path"]
+                    try:
+                        vmware_provider.delete_vmdk(vmdk_path=_vmdk_path)
+                    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)